Setup/autoupdate.sh

```diff
@@ -72,34 +72,21 @@ while true; do
     fi
 
     if [ -f "$ES_AUTO_UPDATE_FILE" ] || [ "$FORCE_CHECK" == true ]; then
-        echo "Getting $ES_VER_FILE_NEW"
-        if [ "$ES_DEV" == true ]; then
-            curl -s -S -o "$ES_VER_FILE_NEW" "$ES_repo_dev_ver"
-        elif [ "$ES_STAGING" == true ]; then
-            curl -s -S -o "$ES_VER_FILE_NEW" "$ES_repo_staging_ver"
-        else
-            curl -s -S -o "$ES_VER_FILE_NEW" "$ES_repo_ver"
+        am_i_leader
+        if [ "$FORCE_CHECK" == true ]; then
+            FORCE_UPDATE="--force"
         fi
-        if [ -f "$ES_VER_FILE" ]; then
-            if [ $(diff "$ES_VER_FILE" "$ES_VER_FILE_NEW" | wc -l) -eq 0 ]; then
-                echo "Your EricomShield System is Up to date"
-            else
-                echo "***************     New version found!"
-                UPDATE=true
-            fi
+
+        if [ "$AM_I_LEADER" == true ]; then
+              if [ "$ES_DEV" == true ]; then
+                $ES_PATH/update.sh update --dev "$FORCE_UPDATE"
+              elif [ "$ES_STAGING" == true ]; then
+                $ES_PATH/update.sh update --staging "$FORCE_UPDATE"
+              else
+                $ES_PATH/update.sh "$FORCE_UPDATE"
```

Comment:
$ES_PATH/update.sh update "$FORCE_UPDATE"

Reply:
Why it's more good?
It's not free if...

Reply:
you made a mistake on line 86 is missing the command _update_